src/reducers/versions.spec.tsx

```diff
@@ -109,6 +109,47 @@ describe(__filename, () => {
       );
     });
 
+    it('adds a path to expandedPaths', () => {
+      const version = fakeVersion;
+      let state = reducer(undefined, actions.loadVersionInfo({ version }));
+
+      expect(state).toHaveProperty(
+        `versionInfo.${version.id}.expandedPaths`,
+        [],
+      );
+
+      const selectedPath = 'new/selected/path';
+      state = reducer(
+        state,
+        actions.toggleExpandedPath({ selectedPath, versionId: version.id }),
+      );
+
+      expect(state).toHaveProperty(`versionInfo.${version.id}.expandedPaths`, [
+        selectedPath,
+      ]);
+    });
+
+    it('removes a path from expandedPaths', () => {
```

Comment:
I suggest one more test that adds two paths, removes one path, and makes sure the first is still in `expandedPaths`.